custom/icds_reports/utils.py

```diff
@@ -44,7 +44,7 @@ def __init__(self, domain, filters, report_id):
         self.report_config = report_config
 
     def data(self):
-        return self.report_config.get_data()
+        return self.report_config.data_source._get_data().values()
```

Comment:
I did some digging and I think this report relies on the person cases. As of right now there are 25 million rows in that table and there will be more as the project goes on.

From my understanding this will call _get_data without any pagination and it will try  to load all 25 million rows in memory which won't work.